libs/telemetry/include/telemetry/registry.hpp

```diff
@@ -74,6 +72,10 @@ class Registry
   using Mutex          = std::mutex;
   using LockGuard      = std::lock_guard<std::mutex>;
 
+  // Construction / Destruction
```

Comment:
As we just talked about - I would revert this particular change

Reply:
ok